doc/decoding/going_further.rst

```diff
@@ -53,35 +49,28 @@ of cross-validation.
 
 You can change many parameters of the cross_validation here, for example:
 
-* use a different cross-validation scheme, for example LeaveOneGroupOut()
+* use a different cross-validation scheme, for example :class:`sklearn.model_selection.LeaveOneGroupOut`.
 
-* speed up the computation by using n_jobs=-1, which will spread the
-   computation equally across all processors.
+* speed up the computation by using `n_jobs=-1`, which will spread the computation equally across all processors.
 
-* use a different scoring function, as a keyword or imported from scikit-learn
-   such as `scoring='roc_auc'`
+* use a different scoring function, as a keyword or imported from scikit-learn such as `scoring='roc_auc'`.
```

Comment:
Just curious: what is the reason for this change ?

Reply:
@bthirion I just added a period at the end of the line to be consistent with lines above, and improved the rendering:

Before:

![before](https://user-images.githubusercontent.com/2639645/123405326-fca82780-d5a9-11eb-8794-cc3c7f2bb2a3.png)


After:

![after](https://user-images.githubusercontent.com/2639645/123405341-00d44500-d5aa-11eb-95f8-b5602befcbe0.png)

Reply:
OK, thx.